src/Microsoft.Azure.WebJobs.Host/Scale/ITargetScaler.cs

```diff
@@ -0,0 +1,27 @@
+﻿// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the MIT License. See License.txt in the project root for license information.
+
+using System.Threading.Tasks;
+
+namespace Microsoft.Azure.WebJobs.Host.Scale
+{
+    /// <summary>
+    /// Interface defining a target scaler that can participate in Azure Functions scaling decisions by
```

Comment:
"Interface defining an Azure Functions scaler that makes scale decisions based on current event source metrics and function concurrency."